usb/storage.c

```diff
@@ -180,14 +180,16 @@ static uint8_t transaction(usb_device_t *dev, command_block_wrapper_t *cbw, uint
     return ret;
   }
 
-  if (cbw->bmCBWFlags & STORAGE_CMD_DIR_IN)
-    info->last_error = usb_in_transfer(dev, &(info->ep[STORAGE_EP_IN]), &size, readbuf);
-  else
-    info->last_error = usb_out_transfer(dev, &(info->ep[STORAGE_EP_OUT]), size, writebuf);
-
-  if(handle_usb_error(dev, (cbw->bmCBWFlags & STORAGE_CMD_DIR_IN) ? STORAGE_EP_IN: STORAGE_EP_OUT)) {
-    storage_debugf("response failed");
-    return STORAGE_ERR_GENERAL_USB_ERROR;
+  if(cbw->CBWCB[0] != SCSI_CMD_TEST_UNIT_READY) {
```

Comment:
Maybe this one could be simply if (size) { ....
One less command specific condition.

Reply:
Yes, thanks. This does the same. PR updated.